network-api/networkapi/buyersguide/templates/about/methodology.html

```diff
@@ -1,171 +1,253 @@
+        which could result in a compromised device.
+        {% endblocktrans %}
       </p>
     </section>
 
     <section aria-labelledby="vulnerability-management">
-      <p id="vulnerability-management"><b>Vulnerability management</b></p>
+      <p id="vulnerability-management"><b>{% trans "Vulnerability management" %}</b></p>
       <p class="body">
-        The vendor must have a system in place to manage vulnerabilities in the product. This must also include a point of contact for reporting vulnerabilities or a bug bounty program. This ensures that vendors are actively managing vulnerabilities throughout the product’s lifecycle. 
+        {% blocktrans trimmed %}
+        The vendor must have a system in place to manage vulnerabilities in the product. This must also include a point
+        of contact for reporting vulnerabilities or a bug bounty program. This ensures that vendors are actively managing
+        vulnerabilities throughout the product’s lifecycle.
+        {% endblocktrans %}
       </p>
     </section>
 
     <section aria-labelledby="privacy-policy">
-      <p id="privacy-policy"><b>Privacy Policy</b></p>
+      <p id="privacy-policy"><b>{% trans "Privacy Policy" %}</b></p>
       <p class="body">
-        The product must have a privacy policy and/or another privacy page that applies to the device, app, or service we are evaluating. 
+        {% blocktrans trimmed %}
+        The product must have a privacy policy and/or another privacy page that applies to the device, app, or service
+        we are evaluating.
+        {% endblocktrans %}
       </p>
     </section>
   </article>
 
   <article class="mt-5 mb-4 about-privacy">
     <h3>
-      Privacy
+      {% trans "Privacy" %}
     </h3>
 
     <p>
-      We evaluated the privacy documentation — which included privacy policies, privacy pages, FAQs – for each company selling these products to determine (1) how data is shared with third parties, (2) whether or not users could request deletion of their data, and (3) the accessibility/readability of the privacy information. Where relevant, we also looked at whether (4) biometric data is collected, and (5) parental controls are in place.
+      {% blocktrans trimmed %}
+      We evaluated the privacy documentation — which included privacy policies, privacy pages, FAQs – for each company
+      selling these products to determine (1) how data is shared with third parties, (2) whether or not users could
+      request deletion of their data, and (3) the accessibility/readability of the privacy information. Where relevant,
+      we also looked at whether (4) biometric data is collected, and (5) parental controls are in place.
+      {% endblocktrans %}
     </p>
 
     <section aria-labelledby="share-data">
-      <p id="share-data"><b>1) How does it share data?</b></p>
+      <p id="share-data"><b>{% trans "1) How does it share data?" %}</b></p>
```

Comment:
do these need context notes for the fact that `1)` etc are enumerations?

Reply:
I don’t think so, that’s pretty much self-explanatory